src/js/models/v2/aboutTheData/BaseAgencyRow.js

```diff
@@ -20,6 +20,9 @@ const BaseAgencyRow = {
         this.tasTotals = data.tas_account_discrepancies_totals;
         this._federalTotal = data.federalTotal;
     },
+    get code() {
```

Comment:
Since there's no re-formatting in the getter, we could just change line 13 to `this.code`.